app/controllers/application_controller.rb

```diff
@@ -11,7 +11,18 @@ def current_user
   end
   helper_method :current_user
 
-  private
+  def require_admin
+    unless admin?
+      redirect_to user_path(current_user), alert: I18n.t(
+        "users.index.require_admin"
+      )
+    end
+  end
+
+  def admin?
+    current_user.admin == true
```

Comment:
This can change to `current_user.admin?`

Reply:
duh! of course, thank you done in sha: 1ea0d5e